src/Examples/CoreExample/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:53646/",
+      "applicationUrl": "http://localhost:55555/",
```

Comment:
You also have to update the port in `"CoreExample": {"applicationUrl": }`

Reply:
Done